.gitignore

```diff
@@ -12,4 +12,7 @@ bin
 .vscode
 *.blend1
 launchSettings.json
-AssetRipper.SourceGenerated.xml
\ No newline at end of file
+AssetRipper.SourceGenerated.xml
+
+/Ripped
+/AssetRipper.log
```

Comment:
I fail to see why these were necessary. They should always be created in a subfolder of `Bins` or `bin`.

Reply:
I think it creates them in the workdir, while I was working on it, I nearly committed these paths a few times 😅